### ISSUE TITLE
keyboardDismissMode is required

### ISSUE THREAD

**Issue by GianMantuan** [ammarahm-ed/react-native-actions-sheet#103]
As shown in the code, it is required to set the property `keyboardDismissMode`, but in the documentation it is said that is not.

How it is in `types.ts`:

```ts
keyboardDismissMode: 'on-drag' | 'none' | 'interactive'
```

How it should be according to the documentation:
```ts 
keyboardDismissMode?: 'on-drag' | 'none' | 'interactive'
```

**Comment by GianMantuan** [comment]
Fix PR: #104

**Comment by GianMantuan** [comment]
Fix merged to master
